tests/daily_scraper/test_daily_scraper.py

```diff
@@ -6,309 +6,132 @@
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central-l10n/firefox-27.0a1.sv-SE.win32.installer.exe'
-    },
-
-    # -p win32 --branch=mozilla-central --build-id=20130706031213
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'build_id': '20130706031213'},
-    'filename': '2013-07-06-03-12-13-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/07/2013-07-06-03-12-13-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'
-    },
-
-    # -p win32 --branch=mozilla-central --date=2013-07-02
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'date': '2013-07-02'},
-    'filename': '2013-07-02-04-12-13-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/07/2013-07-02-04-12-13-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'
-    },
-
-    # -p win32 --branch=mozilla-central --date=2013-07-02 --build-number=1
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'date': '2013-07-02',
-              'build_number': 1},
-    'filename': '2013-07-02-03-12-13-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/07/2013-07-02-03-12-13-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'
-    },
-
-    # -p win32 --branch=mozilla-central --stub (old format)
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'date': '2013-09-30',
-              'is_stub_installer': True},
-    'filename': '2013-09-30-03-02-04-mozilla-central-firefox-27.0a1.en-US.win32.installer-stub.exe',
-    'url': 'firefox/nightly/2013/09/2013-09-30-03-02-04-mozilla-central/firefox-27.0a1.en-US.win32.installer-stub.exe'
-    },
-
-    # -p win32 --branch=mozilla-central --stub (new format)
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'is_stub_installer': True},
-    'filename': '2013-10-01-03-02-04-mozilla-central-Firefox Installer.en-US.exe',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/Firefox Installer.en-US.exe'
-    },
-
-    # -p win32 --branch=mozilla-aurora
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-aurora'},
-    'filename': '2013-10-01-03-02-04-mozilla-aurora-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-aurora/firefox-27.0a1.en-US.win32.installer.exe'
-    },
-
-    # -p win32 --branch=ux
-    {'args': {'platform': 'win32',
-              'branch': 'ux'},
-    'filename': '2013-10-01-03-02-04-ux-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-ux/firefox-27.0a1.en-US.win32.installer.exe'
-    },
-
-    # --revision 1dbe350b57b1
-    {'args': {'platform': 'win32'},
```

Comment:
There is something wrong with the test. It misses the --revision argument. Given that we have a separate file for that in this folder lets get rid of this test.